src/net/if_.rs

```diff
@@ -17,3 +17,77 @@ pub fn if_nametoindex<P: ?Sized + NixPath>(name: &P) -> Result<c_uint> {
         Ok(if_index)
     }
 }
+
+libc_bitflags!(
+    /// Standard interface flags, used by `getifaddrs`
+    pub struct InterfaceFlags: libc::c_int {
```

Comment:
It looks like you only included Linux's interface flags.  Could you please add, at a minimum, FreeBSD's?  That would be `IFF_RUNNING` and `IFF_OACTIVE` (yes, they're defined in FreeBSD, but currently missing from libc), as well as `IFF_SIMPLEX`, `IFF_CANTCONFIG`, `IFF_PPROMISC`, `IFF_MONITOR`, `IFF_STATICARP`, `IFF_DYING`, `IFF_RENAMING`, `IFF_LINK0`, `IFF_LINK1`, and `IFF_LINK2`?

Reply:
Hm.. fails on FreeBSD.

@asomers Want to wait for a `libc` release, or why did you ask for stuff you know isn't in `libc`, yet?

Reply:
No need to wait for a release since nix's master branch depends on libc's master branch.  Just open a pull request to add the missing constants to libc.  We have to do it all the time for nix.